luigi/tools/range.py

```diff
@@ -460,6 +460,11 @@ def finite_datetimes(self, finite_start, finite_stop):
         """
         Simply returns the points in time that correspond to a whole number of minutes intervals.
         """
+        # Validate that the minutes_interval can divide 60 and it is greater than 0.
+        if self.minutes_interval <= 0:
```

Comment:
While you're at it, can you change this to

`if not (0 < self.minutes_interval < 60)`